js/runner_test.go

```diff
@@ -466,8 +468,8 @@ func TestRunnerIntegrationImports(t *testing.T) {
 			name, data := name, data
 			t.Run(name, func(t *testing.T) {
 				r1, err := getSimpleRunner(data.filename, fmt.Sprintf(`
-					import hi from "%s";
-					export default function() {
+					var hi = require("%s").default;
```

Comment:
I need to look into this ...

Reply:
okay so ... basically setting `default` to something only works if babel helps us otherwise we should just say `exports  = "hi!"` if we wont' be using babel.  
much better explanation [here](https://2ality.com/2015/12/babel-commonjs.html). I don't think we need to do anything until we start supporting import/export ouself or through goja :D